src/pybind/mgr/rgw/module.py

```diff
@@ -99,25 +103,78 @@ def _cmd_rgw_admin(self, params: Sequence[str]):
 
     @CLICommand('rgw realm bootstrap', perm='rw')
     def _cmd_rgw_realm_bootstrap(self,
-                                 realm_name : Optional[str] = None,
+                                 realm_name: Optional[str] = None,
                                  zonegroup_name: Optional[str] = None,
                                  zone_name: Optional[str] = None,
                                  endpoints: Optional[str] = None,
                                  sys_uid: Optional[str] = None,
                                  uid: Optional[str] = None,
-                                 start_radosgw: Optional[bool] = True):
+                                 start_radosgw: Optional[bool] = True,
+                                 zonegroup_is_master: Optional[bool] = True,
+                                 zone_is_master: Optional[bool] = True):
```

Comment:
are we going to have any docs on what these two `is_master`  or the `start_radosgw` settings do?

Reply:
code has changed we do not provide the user the possibility to configured master (we use master by default).